gemfileparser/__init__.py

```diff
@@ -124,29 +118,43 @@ def parse_line(self, line):
             line = unicode(line)
         except NameError:
             pass
-        dep = self.Dependency()
-        dep.group = GemfileParser.global_group
-        if not self.appname:
-            dep.parent = []
-        else:
+        linefile = io.StringIO(line)    # csv requires a file object
+        for line in csv.reader(linefile, delimiter=','):
```

Comment:
Can you remind my the benefits of using a CSV module here?

Reply:
I think @balasankarc used this CSV module here because dependency contains name and different version numbers and they are seperated by comma and they have to split by comma and if we use something else then to remove extra character the code will become more complex. CSV module might have make this easy to remove extra characters. I guess this might be the only good solution to use CSV module here.